RDF-Pipeline/t/tests/0049_DependsOn_clock_site/setup-files/node/pipeline.ttl

```diff
@@ -5,5 +5,5 @@
 @prefix : <http://localhost/node/> .
 
 :hello a p:FileNode ;
-  p:dependsOn <http://www.time.gov/timezone.cgi?Eastern/d/-5> .
+  p:dependsOn <http://www.time.gov> .
```

Comment:
Aha!  Here's one that should work: http://tycho.usno.navy.mil/cgi-bin/timer.pl
It sets both Last-Modified and ETag